### PR TITLE
coverage-text: do not use colors if --colors=never

### PR DESCRIPTION
**Pull request by gkralik**
Don't use colors when using `--coverage-text` along with `--colors=never`.

See #1771 and #1892 

(replaces #1906)

### CODE REVIEW

File: src/TextUI/TestRunner.php [hunk 1]
```diff
@@ -490,6 +490,9 @@ class_exists($arguments['printer'], false)) {
                 if ($arguments['coverageText'] == 'php://stdout') {
                     $outputStream = $this->printer;
                     $colors       = $arguments['colors'];
+                    if($colors == PHPUnit_TextUI_ResultPrinter::COLOR_NEVER) {
```
Comment (GrahamCampbell) [inline src/TextUI/TestRunner.php]: why not just `$colors       = $arguments['colors'] && !PHPUnit_TextUI_ResultPrinter::COLOR_NEVER`?

Comment (GrahamCampbell) [inline src/TextUI/TestRunner.php]: just thinking about avoiding a double assignment - maybe your way is more readable though :)

Comment (gkralik) [inline src/TextUI/TestRunner.php]: yeah, just thought about the same thing... will change it

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request closed without merging**